jest.web.config.js

```diff
@@ -4,7 +4,7 @@ const { excludeCollectCoverageFrom } = require('./jest.excludeCollectCoverageFro
 module.exports = {
   ...base,
   preset: '',
-  testEnvironment: './jest/fixJestEnvironmentJsdom.ts',
+  testEnvironment: process.env.RUN_ALLURE === 'true' ? 'allure-jest/jsdom' : 'jsdom',
```

Comment:
j'ai l'impression qu'on fait du code mort

soit il faut supprimer le fichier `./jest/fixJestEnvironmentJsdom.ts`

soit il faut l'utiliser

```suggestion
  testEnvironment: process.env.RUN_ALLURE === 'true' ? 'allure-jest/jsdom' : './jest/fixJestEnvironmentJsdom.ts',
```